.travis.yml

```diff
@@ -1,18 +1,12 @@
 language: php
 
-sudo: false
-
-# whitelist for building branches
-branches:
-  only:
-    - master
-    - stable
-
```

Comment:
Could you revert these changes? Or can you explain why you've chosen to remove these lines?

Reply:
These lines cause travis to only. trigger on master or on stable branches, meaning merge requests wont be tested by travis automatically.